the nghttp3 build does not even attempt to detect whether we provide netinet/in.h

diff -wpruN --no-dereference '--exclude=*.orig' a~/deps/ngtcp2/nghttp3/lib/nghttp3_conv.h a/deps/ngtcp2/nghttp3/lib/nghttp3_conv.h
--- a~/deps/ngtcp2/nghttp3/lib/nghttp3_conv.h	1970-01-01 00:00:00
+++ a/deps/ngtcp2/nghttp3/lib/nghttp3_conv.h	1970-01-01 00:00:00
@@ -34,9 +34,7 @@
 #  include <arpa/inet.h>
 #endif /* HAVE_ARPA_INET_H */
 
-#ifdef HAVE_NETINET_IN_H
 #  include <netinet/in.h>
-#endif /* HAVE_NETINET_IN_H */
 
 #ifdef HAVE_BYTESWAP_H
 #  include <byteswap.h>
